mesonbuild/interpreter/interpreter.py

```diff
@@ -1720,6 +1723,7 @@ def find_program_fallback(self, fallback: str, args: T.List[mesonlib.FileOrStrin
         REQUIRED_KW,
         KwargInfo('dirs', ContainerTypeInfo(list, str), default=[], listify=True, since='0.53.0'),
         KwargInfo('version', ContainerTypeInfo(list, str), default=[], listify=True, since='0.52.0'),
+        DEFAULT_OPTIONS.evolve(since='1.3.0'), allow_unknown=True
```

Comment:
Why adding `allow_unknown=True`?

Reply:
I copied the statement from the kwargs of the dependency() function

Reply:
Well, those are two different arguments, only one of which is relevant here... :)

Reply:
Didn't realize they were unrelated. I removed allow_unknown :)